resources/dicts/thoughts/alive/apprentice.json

```diff
@@ -141,18 +152,22 @@
             "Asks r_c what it's like to be an elder",
             "Is doing {PRONOUN/m_c/poss} best to keep r_c happy",
             "Is bringing r_c a nice piece of prey",
-            "Is complaining about picking ticks out of the elders' pelts"
+            "Is complaining about picking ticks out of the elders' pelts",
+            "Is offering an elder to do tick duty",
```

Comment:
should reword to "Is offering to do tick duty for an elder", otherwise it sounds like they're trying to make an elder do tick duty